packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -561,6 +561,47 @@ describe("DropdownComponent", () => {
 				});
 			});
 
+			describe("DropDown properties", () => {
```

Comment:
Could you please change the description for something more related to what you will test ? 😊